mi-scheduler/base/cronjob.yaml

```diff
@@ -133,8 +131,86 @@ spec:
                       key: github-access-token
                 - name: SCHEDULE_KEBECHET_ANALYSIS
                   value: "1"
-                - name: SCHEDULE_GH_REPO_ANALYSIS
+                - name: THOTH_ADJUST_LOGGING
+                  value: 'thoth.mi-scheduler:INFO'
+          restartPolicy: OnFailure
+---
+kind: CronJob
```

Comment:
is this a cronjob or do we want it to be an ArgoCronjobWorkflow? @harshad16

Reply:
do we want to join kebechet's `merge` and `analyse` into the one ArgoCronjobWorkflow? Or what's the use case here?

Reply:
it was more a general question, CronJob is good ;)